src/experimental/react/controllers/animation-map-controller.js

```diff
@@ -193,13 +193,12 @@ export default class AnimationMapController extends PureComponent {
   }
 
   _updateViewport() {
-    const animatedViewport = Object.assign({}, this.state.animationEndViewport);
     const t = this.props.viewportAnimationEasingFunc(this.state.animationT);
-    for (const p of VIEWPORT_ANIMATE_PROPS) {
-      const startValue = this.state.animationStartViewport[p];
-      const endValue = this.state.animationEndViewport[p];
-      animatedViewport[p] = this._interpolate(startValue, endValue, t);
-    }
+    const animatedViewport = this.props.viewportAnimationFunc(
+      this.state.animationStartViewport,
+      this.state.animationEndViewport,
+      t
```

Comment:
Let's avoid abbreviated identifiers. Typing out `time` will increase readability and the minifier will reduce it back to `t` in production code.